### PR TITLE
docs: update pod-priority-preemption.md

### LINKED ISSUE #37740: [ko] Missing one line translation website/content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption
**This is a Feature Request**

**What would you like to be added**
Translate untranslated line in
`website/content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption`
into Korean.

**Why is this needed**
Missing one line translation : 
`the scheduler always tries the "nominated Node" before iterating over any other nodes.`

**Page**
https://kubernetes.io/ko/docs/concepts/scheduling-eviction/pod-priority-preemption/#%EC%82%AC%EC%9A%A9%EC%9E%90-%EB%85%B8%EC%B6%9C-%EC%A0%95%EB%B3%B4

/language ko

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
### Fixes

Closes #37740

### Changes proposed

Translated `the scheduler always tries the "nominated Node" before iterating over any other nodes.` to Korean. 

File path `website/content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption`

### CODE REVIEW

File: content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ spec:
 정보를 제공한다.
 
 파드 P는 반드시 "지정된 노드"로 스케줄링되지는 않는다.
-The scheduler always tries the "nominated Node" before iterating over any other nodes.
+스케줄러는 다른 노드를 반복하기 전에 항상 "nominated Node"를 시도합니다.
```
Comment (Aut0R3V) [inline content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md:206]: Looks good to me!

File: content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ spec:
...
 스케줄러는 다른 노드에 스케줄링을 시도하기 전에 항상 "지정된 노드"부터 시도한다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md:207]: Hi @Pradumnasaraf 
Thank you for the contribution!

As a native Korean, the translated sentence is not natural to me. (is it machine translation?) 
https://github.com/kubernetes/website/issues/37740#issuecomment-1304755015
and, seem not compatible with the Korean localization style https://kubernetes.io/ko/docs/contribute/localization_ko/ 

But, I was realized the untranslated English sentence was actually translated in the next line.
So, we can just remove the 206 line (the English sentence)
```suggestion
스케줄러는 다른 노드에 스케줄링을 시도하기 전에 항상 "지정된 노드"부터 시도한다.
```

Korean localization uses own development branch. 
Please change base of this PR to [dev-1.25-ko.1](https://github.com/kubernetes/website/tree/dev-1.25-ko.1) which is the current development branch for Korean localization.

Comment (Pradumnasaraf) [reply content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: Hey,
Yes, I take some help from machine translation, as I am learning Korean. Thank you for the review

**Comment by Pradumnasaraf** [comment]
Need to new Pr as the base PR branch should also be the [dev-1.25-ko.1](https://github.com/kubernetes/website/tree/dev-1.25-ko.1)

File: content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md (hunk 2 above)
Comment (seokho-son) [reply content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: @Pradumnasaraf 
Good to know you are learning Korean :)
Thanks for the contribution~!

**Comment by jihoon-seo** [comment]
/remove-area blog release-eng
/remove-language de en es fr hi id it ja pl pt vi zh
/remove-sig release

**Pull request closed without merging**